_config.yml

```diff
@@ -94,137 +139,6 @@ flink_releases:
           url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-json/1.9.2/flink-json-1.9.2.jar
           asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-json/1.9.2/flink-json-1.9.2.jar.asc
           sha_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-json/1.9.2/flink-json-1.9.2.jar.sha1
-  -
-      version_short: 1.8
```

Comment:
we could move this removal into a separate commit

Reply:
done